test/functional/http/post/3.votes.js

```diff
@@ -0,0 +1,437 @@
+'use strict';
+
+var node = require('../../../node');
+var shared = require('../../shared');
+var constants = require('../../../../helpers/constants');
+
+var sendTransactionPromise = require('../../../common/apiHelpers').sendTransactionPromise;
+var creditAccountPromise = require('../../../common/apiHelpers').creditAccountPromise;
+var registerDelegatePromise = require('../../../common/apiHelpers').registerDelegatePromise;
+var waitForBlocksPromise = node.Promise.promisify(node.waitForBlocks);
+
+describe('POST /api/transactions (type 3) votes', function () {
+
+	var badTransactions = [];
+	var goodTransactions = [];
+	var badTransactionsEnforcement = [];
+	var goodTransactionsEnforcement = [];
+
+	var account = node.randomAccount();
+	var accountNoFunds = node.randomAccount();
+	var accountMinimalFunds = node.randomAccount();
+	var accountDuplicate = node.randomAccount();
+
+	var account33 = node.randomAccount();
+	var delegates33 = [];
+
+	var account101 = node.randomAccount();
+	var delegates101 = [];
+
+	var transaction;
+
+	before(function () {
+		// Crediting accounts and registering delegates
+		var promises = [];
+		promises.push(creditAccountPromise(account.address, 1000 * node.normalizer));
+		promises.push(creditAccountPromise(accountMinimalFunds.address, constants.fees.vote));
+		promises.push(creditAccountPromise(node.eAccount.address, 1000 * node.normalizer));
+		promises.push(creditAccountPromise(account33.address, 1000 * node.normalizer));
+		promises.push(creditAccountPromise(account101.address, 1000 * node.normalizer));
+		promises.push(creditAccountPromise(accountDuplicate.address, constants.fees.vote * 2));
+
+		return node.Promise.all(promises).then(function (results) {
+			results.forEach(function (res) {
+				node.expect(res).to.have.property('success').to.be.ok;
+				node.expect(res).to.have.property('transactionId').that.is.not.empty;
+			});
+		})
+			.then(function (res) {
+				var promisesCredits33 = [];
+				for (var i = 0; i < 33; i++) {
```

Comment:
Try to name the variables instead of using magic numbers. 
`var MAX_VOTES_IN_ONE_TRANSACTION = 33;`